src/files/FilesPage.js

```diff
@@ -1,2 +1,114 @@
 import React from 'react'
-export default () => <h1 data-id='title'>Files</h1>
+import PropTypes from 'prop-types'
+import { connect } from 'redux-bundler-react'
+import Breadcrumbs from './breadcrumbs/Breadcrumbs'
+import FilesList from './files-list/FilesList'
+import FilePreview from './file-preview/FilePreview'
+import FileInput from './file-input/FileInput'
+
+const action = (name) => {
+  return (...args) => {
+    console.log(name, args)
+  }
+}
+
+const empty = (
+  <div>
+    <h2>It seems a bit lonely here :(</h2>
```

Comment:
@akrych when you get time, could you create some ideas for a "you have no files yet, why not add some" page for the files view? It could use some of the space to introduce some IPFS or at least some of the issues around "they are only on your machine until a peer fetches them from you", and "this is p2p. this is not a cloud"